scripts/release-branch.js

```diff
@@ -0,0 +1,146 @@
+// Added shelljs to use maybe in the script "npm install shelljs"
+// https://github.com/shelljs/shelljs
+const shell = require('shelljs');
+
+
+const Octokit = require("@octokit/rest");
+const octokit = new Octokit();
```

Comment:
Need to use Auth here: 
```
const octokit = new Octokit({auth: process.env.DANGER_GITHUB_API_TOKEN});
```
I'm not actually sure what format the token is in right now - it should be `token XXX` (note the use of the word token).  See this page for auth details: https://octokit.github.io/rest.js/#authentication